owlbot.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Google LLC
```

Comment:
nit: can we make this a git mv if we're keeping the license year?

Reply:
Apparently git does not fully support this so we'll keep the year from the original synth.py.